backend/app/graphql/onboarding_request.py

```diff
@@ -31,7 +28,69 @@ class OnboardingRequest(graphene.ObjectType):
     status = graphene.String()
 
 
+# Queries
+
+# Return object for queries
+class GetOnboardingRequest(Query):
+    contact_name = graphene.String()
+    contact_email = graphene.String()
+    contact_phone = graphene.String()
+    role = graphene.String()
+    date_submitted = graphene.DateTime()
+    status = graphene.String()
+
+
+class OnboardingRequestQueries(QueryList):
+    getAllOnboardingRequests = graphene.List(
+        GetOnboardingRequest,
+        number=graphene.Int(default_value=5),
+        offset=graphene.Int(default_value=0),
+        role=graphene.String(default_value=""),
+        status=graphene.String(default_value=""),
+    )
+    getOnboardingRequestById = graphene.List(
+        GetOnboardingRequest,
+        id=graphene.String(),
+    )
+
+    def resolve_getAllOnboardingRequests(self, info, number, offset, role, status):
+        requests = services["onboarding_request_service"].get_all_onboarding_requests(
+            number, offset, role, status,
+        )
+        return [
+            *map(
```

Comment:
BTW, I did some digging, and I think you might not need to use `map` here at all! I believe that `graphene` should auto-convert to the correct type if you return the request directly. So instead of a map, you would just `return requests`.